hindi/joke/joke.md

```diff
@@ -1,3 +1,4 @@
+[here](https://github.com/saksham0037/create-your-own-adventure/blob/edit/hindi/Akshay_Kumar/akshay_kumar.md)
```

Comment:
This link seems to be an absolute link to GitHub rather than a relative link.  If you are linking to another page in the create-your-own-adventure repository, please use relative links to do so.  See the README of this repository for more information.